streams/src/main/java/org/apache/kafka/streams/state/WindowStore.java

```diff
@@ -26,7 +27,10 @@
 import static org.apache.kafka.streams.internals.ApiUtils.prepareMillisCheckFailMsgPrefix;
 
 /**
- * A windowed store interface extending {@link StateStore}.
+ * Interface for storing the aggregated values of fixed-size time windows.
+ * <p>
+ * The key is internally represented as {@link Windowed Windowed&lt;K&gt;} that comprises the plain key
```

Comment:
Ditto here. I'd rather update the javadoc of `Windowed` and just reference it from WindowStore / TimestampedWindowStore. Also I think we do not need to talk about impl key representation since it is internal.